content/publications/index.md

```diff
@@ -12,6 +12,45 @@ A list of some of the papers directly from our group, with some short explanatio
 
 {{< toc >}}
 
+## 2023
+
+### Accurate and fast graph-based pangenome annotation and clustering with ggCaller
+
+Samuel T. Horsfield, Nicholas J. Croucher and John A. Lees (2023). **Accurate and fast graph-based pangenome annotation and clustering with ggCaller**
+*bioRxiv*
+
+https://www.biorxiv.org/content/10.1101/2023.01.24.524926v1
+
+Annotation of bacterial pangenomes is typically done one at a time. Each input assembly
+is annotated by calling and scoring ORFs, and aligning all of these against databases
```

Comment:
It may be a good idea just to highlight what ORF and COG stand for for the uninitiated.